R/accounts.R

```diff
@@ -51,7 +51,7 @@ accounts <- function(server = NULL) {
 #'
 #' @family Account functions
 #' @export
-connectApiUser <- function(account = NULL, server = NULL, apiKey = NULL, quiet = FALSE) {
+connectApiUser <- function(account = NULL, server = NULL, apiKey, quiet = FALSE) {
```

Comment:
I'm pretty sure this is required; it certainly doesn't work without it.

Reply:
Agreed. This is part of the public package API; NEWS?